ksqldb-engine/src/main/java/io/confluent/ksql/engine/KsqlEngine.java

```diff
@@ -177,7 +176,6 @@ public KsqlEngine(
         1000,
         TimeUnit.MILLISECONDS
     );
-    this.ksqlConfig = Objects.requireNonNull(ksqlConfig, "ksqlConfig");
```

Comment:
Sorry for the confusion. This is not related to this PR but to the one I branched off of https://github.com/confluentinc/ksql/pull/8334. We removed it because it is not owned by the context instead of the ksqlengine.